changes/noissue.79.feature.rst

```diff
@@ -0,0 +1 @@
+Test: Migrated to using pytest log control for tests.
```

Comment:
Suggest add comment "as documented in docs/development.rst".

Reply:
done